### ISSUE TITLE
PEAR.Functions.FunctionCallSignature should ignore comments

### ISSUE THREAD

**Issue by Flimm** [squizlabs/PHP_CodeSniffer#1616]
I have an example PHP file, with these contents:

```php
<?php

example(
    $one,
    $two,
    $three,
//    $four,
//    $five,
    $six,
    $seven
);
```

When I run phpcs, I get this output:

```
$ phpcs -s example.php

FILE: example.php
----------------------------------------------------------------------------------------------------------------------------------------------
FOUND 2 ERRORS AFFECTING 2 LINES
----------------------------------------------------------------------------------------------------------------------------------------------
 7 | ERROR | [x] Multi-line function call not indented correctly; expected 4 spaces but found 0 (PEAR.Functions.FunctionCallSignature.Indent)
 8 | ERROR | [x] Multi-line function call not indented correctly; expected 4 spaces but found 0 (PEAR.Functions.FunctionCallSignature.Indent)
----------------------------------------------------------------------------------------------------------------------------------------------
PHPCBF CAN FIX THE 2 MARKED SNIFF VIOLATIONS AUTOMATICALLY
----------------------------------------------------------------------------------------------------------------------------------------------
```

`PEAR.Functions.FunctionCallSignature.Indent` does not seem to like the fact that the lines containing `$four` and `$five` were commented out by inserting `//` at the beginning of the line, it insists that the comments be indented too. If I run `phpcbf`, this is how it corrects the file:

```php
<?php

example(
    $one,
    $two,
    $three,
    //    $four,
    //    $five,
    $six,
    $seven
);
```

I would like to find a way to get `PEAR.Functions.FunctionCallSignature.Indent` to ignore lines that contain only comments. This commenting style is used by some editors commenting shortcuts, and I don't want to require users to switch editors or spend time messing with their editor settings to make it comment in an indented style, I want to allow comments like this no matter where they are.

**Comment by gsherwood** [comment]
This sniff in the PEAR standard is unable to ignore comments, and no other included sniffs will check indentation and ignore comment lines either.

I'll put this down as a feature request to have this sniff contain an option that allows commented lines to be ignored.

**Comment by gsherwood** [comment]
Closing due to lack of additional interest
